### PR TITLE
[ETFE-744] Add check answers item page

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -35,7 +35,7 @@ lazy val root = (project in file("."))
     PlayKeys.playDefaultPort := 8313,
     ScoverageKeys.coverageExcludedFiles := "<empty>;Reverse.*;.*handlers.*;.*components.*;" +
       ".*Routes.*;.*viewmodels.govuk.*;.*testOnly.*;",
-    ScoverageKeys.coverageMinimumStmtTotal := 85,
+    ScoverageKeys.coverageMinimumStmtTotal := 95,
```
Comment (james-work-account) [inline build.sbt:38]: 😎

**Comment by james-work-account** [comment]
Wait for #46 before merging this - I'll rebase off main before opening up again once that's in

File: app/viewmodels/checkAnswers/ShortageOrExcessItemSummary.scala [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+  private def shortageOrExcessInformationRow(idx: Int)(implicit request: DataRequest[_], messages: Messages): Option[SummaryListRow] = {
+    if (isShortageOrExcess(idx)) {
+      request.userAnswers.get(ItemShortageOrExcessPage(idx)).map(
+        shortageOrExcess =>
+          shortageOrExcess.additionalInfo match {
+            case Some(value) if value != "" =>
+              SummaryListRowViewModel(
+                key = s"${ItemShortageOrExcessPage(idx)}.checkYourAnswers.additionalInfo.label",
+                value = ValueViewModel(Text(value)),
```
Comment (stephenkward) [inline app/viewmodels/checkAnswers/ShortageOrExcessItemSummary.scala:94]: shortage or excess reason value that is displayed on the check answers item page needs to start with uppercase

**Review by stephenkward — CHANGES_REQUESTED** [review]
added couple of comments to PR

**Review by stephenkward — APPROVED** [review]

**Pull request merged**